broadinstitute/single_cell_portal_core
# adding MIXPANEL SECRET to nginx and docker env (SCP-4327)

This correctly loads the new MIXPANEL_SECRET into the server environment

TO TEST:
<<probably much easier and more accurate to test by just waiting until this is merged and then testing on staging, but you can test this locally by>>
1. load SCP in dockerized mode (via `load_env_secrets.sh`)
2. go to a study, and click 'Usage stats' in 'Settings'
3. confirm the stats load, and no errors
